app/javascript/components/froggo-team-show.vue

```diff
@@ -134,14 +143,32 @@ export default {
         id: this.froggoTeam.id,
       })
         .then(() => {
-          window.location.href = `/froggo_teams/${this.froggoTeam.id}`;
+          this.$store.commit(TEAM_NAME, this.newName);
+        })
+        .catch(() => {
+          this.showMessage('nombre existente');
         });
     },
-    updateSelected(selectedUsers) {
-      this.newUsers = selectedUsers;
+    onItemClicked({ item }) {
+      this.addMember(item);
+    },
+    addMember(user) {
+      if (this.froggoTeamMembers.some(u => u.id === user.id)) {
+        const index = this.oldUsers.findIndex(u => u.id === user.id);
+        this.oldUsers.splice(index, 1);
+      } else {
+        this.newUsers = [...this.newUsers, user];
+      }
```

Comment:
me podi explicar la lógica acá?, no sé si me queda claro qué cosa va en qué arreglo. Quizás podríamos intentar simplificar esta lógica pa ver si queda más "autoexplicativa" y lo mismo para el remove member

Reply:
ya, lo que pasa es que hay una lista this.froggoTeamMembers que viene de back, que tiene todos los miembros que el equipo tiene en la BDD, pero a medida que el usuario quiere ir agregando o sacando usuarios, se van agregando respectivamente en las listas newUsers y oldUsers ... y a final cuando el usuario pone "guardar los cambios" se le manda a back las listas con los nuevos y los viejos y ahí se crean y destruyen las memberships

Reply:
ya buena entonces acá la idea es que si está el usuario como miembro original quiere decir que pasó por old users (solo en front sin perpetuarse a back) pero ya no tiene que ir ahí? En el fondo está volviendo a agregar un usuario que había sacado pero ya no quiere sacar.

Reply:
exactooooo!!! tiene que revisar si está agregando un usuario nuevo o está re agregando un usuario del equipo que antes había sacado